app/views/org_admin/templates/history.html.erb

```diff
@@ -18,10 +18,9 @@
         partial: '/paginable/templates/history',
         controller: 'paginable/templates',
         action: 'history',
-        path_params: { id: template.id },
-        query_params: { sort_field: :version, sort_direction: :desc }, 
+        query_params: { sort_field: 'templates.version', sort_direction: 'desc' }, 
```

Comment:
I remember I placed path_params when I added pagination for this action otherwise how do you know how to filter the history of templates for a specific template id? I think the pagination is broken if that line is removed...